layouts/partials/header.html

```diff
@@ -50,6 +50,12 @@
 <!-- End Google Tag Manager (noscript) -->
 
 <nav class="navbar navbar-fixed-top">
+  <div class="navbar-container navbar-top-section">
+    <span>Looking for Camunda Platform 8? >></span>
+    <a target="_blank" href="https://docs.camunda.io/">Docs</a>
+    <a target="_blank" href="https://signup.camunda.com/accounts?_ga=2.124018762.1966193156.1686668003-663653735.1686668003">Create account</a>
```

Comment:
Based on https://github.com/camunda/camunda-platform-docs/pull/2287, I suspect we'd want to remove the `_ga=...` from this link, but it should be clarified with the Acquisition team.

Reply:
@jpguisa can you adjust this link to `https://signup.camunda.com/accounts?utm_source=docs.camunda.org&utm_medium=referral` please?